lang/CMakeLists.txt

```diff
@@ -222,6 +223,14 @@ if (NOT WIN32)
 		PROPERTY OUTPUT_NAME sclang)
 endif()
 
+if (SC_ABLETON)
```

Comment:
OK -- missed a couple in ./CMakeLists.txt btw. (Because I'm not super-familiar with cmake scripts, I insisted on testing before committing -- and found that changing it in lang/CMakeLists.txt is not sufficient.)

Starting with a fresh build directory, the default cmake output now includes:

```
-- Compiling with Ableton Link support
...

// and cmake .. -L:
SC_ABLETON_LINK:BOOL=ON
```